src/components/Messages/MessageList/styles.scss

```diff
@@ -33,3 +33,86 @@
 		list-style: none;
 	}
 }
+
+.call {
+	position: absolute;
+	top: 0;
+
+	display: flex;
+	flex-direction: column;
+
+	width: 100%;
+	height: 50%;
+	margin-bottom: 5px;
+
+	padding-top: 40px;
+	padding-bottom: 0;
+
+	text-align: center;
+
+	color: #ffffff;
+
+	border: 1px solid black;
+
+	background: #1f2329;
+
+	font-family: Inter;
+	font-size: 15px;
+	font-weight: 600;
+	font-style: normal;
+	line-height: 16px;
+	justify-content: space-between;
+}
+
+.btn {
+	display: flex;
+	flex-direction: row;
+
+	margin-bottom: 1px;
+	padding: 10px;
+	padding-top: 0;
+
+	color: white;
+	justify-content: center;
+}
+
+.avatar {
```

Comment:
On the original design I think the avatar looks a bit bigger? Could you please check?